src/core/transactionmanager.js

```diff
@@ -476,6 +476,11 @@ TransactionManager.prototype._resubmit = function(ledgers_, pending_) {
       }
     }
 
+    if (!transaction.isResubmittable()) {
+      // Allow the transaction to fail via ledger close
```

Comment:
I'm having trouble understanding this comment, could you add more explanation to it?